docs/data/charts/export/export.md

```diff
@@ -18,6 +18,24 @@ The print dialog allows you to print the chart or save it as a PDF, as well as c
 
 {{"demo": "PrintChart.js"}}
 
+## Export as image
+
+The `apiRef` prop also exposes a `exportAsImage` function that can be used to export the chart as an image.
```

Comment:
```suggestion
The `apiRef` prop also exposes a `exportAsImage()` method to export the chart as an image.
```

cf: **Referencing JS identifiers** section in https://mui-org.notion.site/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b